solr/core/src/java/org/apache/solr/blockcache/BlockDirectory.java

```diff
@@ -289,12 +290,12 @@ String getFileCacheName(String name) throws IOException {
 
   private long getFileModified(String name) throws IOException {
     if (in instanceof FSDirectory) {
-      File directory = ((FSDirectory) in).getDirectory().toFile();
-      File file = new File(directory, name);
-      if (!file.exists()) {
+      Path directory = ((FSDirectory) in).getDirectory();
+      Path file = directory.resolve(name);
```

Comment:
Maybe just purely cosmentic, but should this really be `path` instead of `file`?  In all the places?

Reply:
No.  I like 'file' to differentiate from a 'dir'.  'path' is generic if could be either.